### ISSUE TITLE
"no results found for help)" on pacaur -Syu

### ISSUE THREAD

**Issue by jakebailey** [rmarquis/pacaur#318]
Background: I'm on pacaur-git in the AUR.

It looks like pacaur is running "pacman -Sp" with no arguments, which results in it grabbing "help)" as something to search for in the AUR (which it of course can't find).

This seems similar to #267, but I don't have any extra repos added.

Here's a bash -x output, but let me know if you want any more info.

```
+ version=4.2.20.4.ge48600f
+ LC_COLLATE=C
+ TEXTDOMAIN=pacaur
+ TEXTDOMAINDIR=/usr/share/locale
+ source /etc/makepkg.conf
++ DLAGENTS=('ftp::/usr/bin/curl -fC - --ftp-pasv --retry 3 --retry-delay 3 -o %o %u' 'http::/usr/bin/curl -fLC - --retry 3 --retry-delay 3 -o %o %u' 'https::/usr/bin/curl -fLC - --retry 3 --retry-delay 3 -o %o %u' 'rsync::/usr/bin/rsync --no-motd -z %u %o' 'scp::/usr/bin/scp -C %u %o')
++ VCSCLIENTS=('bzr::bzr' 'git::git' 'hg::mercurial' 'svn::subversion')
++ CARCH=x86_64
++ CHOST=x86_64-unknown-linux-gnu
++ CPPFLAGS=-D_FORTIFY_SOURCE=2
++ CFLAGS='-march=core-avx2 -O2 -pipe'
++ CXXFLAGS='-march=core-avx2 -O2 -pipe'
++ LDFLAGS=-Wl,-O1,--sort-common,--as-needed,-z,relro
++ MAKEFLAGS=-j8
++ DEBUG_CFLAGS='-g -fvar-tracking-assignments'
++ DEBUG_CXXFLAGS='-g -fvar-tracking-assignments'
++ BUILDENV=(!distcc color !ccache check !sign)
++ OPTIONS=(strip docs !libtool !staticlibs emptydirs zipman purge !upx !debug)
++ INTEGRITY_CHECK=(md5)
++ STRIP_BINARIES=--strip-all
++ STRIP_SHARED=--strip-unneeded
++ STRIP_STATIC=--strip-debug
++ MAN_DIRS=({usr{,/local}{,/share},opt/*}/{man,info})
++ DOC_DIRS=(usr/{,local/}{,share/}{doc,gtk-doc} opt/*/{doc,gtk-doc})
++ PURGE_TARGETS=(usr/{,share}/info/dir .packlist *.pod)
++ COMPRESSGZ=(gzip -c -f -n)
++ COMPRESSBZ2=(bzip2 -c -f)
++ COMPRESSXZ=(xz -T0 -c -z -)
++ COMPRESSLRZ=(lrzip -q)
++ COMPRESSLZO=(lzop -q)
++ COMPRESSZ=(compress -c -f)
++ PKGEXT=.pkg.tar.xz
++ SRCEXT=.src.tar.gz
+ [[ -r /home/jake/.makepkg.conf ]]
+ tmpdir=/tmp/pacaurtmp-jake
+ builddir=/tmp/pacaurtmp-jake
+ editor=vim
+ editpkgbuild=true
+ editinstall=true
+ fallback=true
+ clean=true
+ cleandevel=true
+ sortbyvotes=true
+ sudoloop=false
+ pacmanbin=pacman
+ [[ -n '' ]]
+ configdir=/etc/xdg/pacaur
+ userconfigdir=/home/jake/.config/pacaur
+ source /etc/xdg/pacaur/config
++ builddir=/home/jake/pacaurtmp
++ editor=vim
++ cleandevel=false
+ [[ -r /home/jake/.config/pacaur/config ]]
+ [[ -d /tmp/pacaurtmp-jake ]]
+ [[ ! -w /tmp/pacaurtmp-jake ]]
+ [[ ! -d /tmp/pacaurtmp-jake ]]
+ [[ ! -d /home/jake/pacaurtmp ]]
+ trap ctrlc INT
+ args=("$@")
+ for i in '"${args[@]}"'
+ [[ -Syu =~ ^-[a-zA-Z0-9] ]]
+ opts+=("$i")
+ count=0
+ [[ -n -Syu ]]
+ getopts sidmykufecqrahvVDQRSTUbglnoptw-: OPT
+ pacmanarg+=("-$OPT")
+ case "$OPT" in
+ pacS=1
+ operation=sync
+ coweropts+=("-f")
+ [[ -Syu =~ w ]]
+ installpkg=true
+ [[ -Syu =~ g ]]
+ [[ -Syu =~ l ]]
+ [[ -Syu =~ p ]]
+ getopts sidmykufecqrahvVDQRSTUbglnoptw-: OPT
+ pacmanarg+=("-$OPT")
+ case "$OPT" in
+ [[ -n 1 ]]
+ operation=sync
+ refresh=true
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n 1 ]]
+ continue
+ getopts sidmykufecqrahvVDQRSTUbglnoptw-: OPT
+ pacmanarg+=("-$OPT")
+ case "$OPT" in
+ [[ -n '' ]]
+ [[ -n 1 ]]
+ operation=sync
+ upgrade=true
+ coweropts+=("-f")
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n 1 ]]
+ continue
+ getopts sidmykufecqrahvVDQRSTUbglnoptw-: OPT
+ [[ -z '' ]]
+ break
++ grep '^Color' /etc/pacman.conf
+ [[ -n Color ]]
+ [[ '' != \n\e\v\e\r ]]
+ [[ '' = \a\l\w\a\y\s ]]
+ coweropts+=("--color=auto")
+ reset='\e[0m'
+ colorR='\e[1;31m'
+ colorG='\e[1;32m'
+ colorY='\e[1;33m'
+ colorB='\e[1;34m'
+ colorM='\e[1;35m'
+ colorW='\e[1;39m'
+ pacmanarg=(${pacmanarg[@]/--/})
+ pacmanarg=(${pacmanarg[@]/-r/})
+ pacmanarg=(${pacmanarg[@]/-a/})
+ [[ sync = sync ]]
+ [[ ! -n '' ]]
+ [[ ! -n '' ]]
+ [[ ! -n '' ]]
+ [[ 1000 -eq 0 ]]
+ [[ -n 1 ]]
+ pacmanarg=(${pacmanarg[@]/-e/})
+ [[ -n 1 ]]
+ [[ -n '' ]]
+ [[ -n 1 ]]
+ [[ -n '' ]]
+ [[ ! -n vim ]]
+ [[ ! -w /home/jake/pacaurtmp ]]
+ [[ -z '' ]]
+ [[ sync = download ]]
+ [[ sync = sync ]]
+ [[ ! -n true ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ case $operation in
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n true ]]
+ [[ ! -n '' ]]
+ ClassifyPkgs
+ [[ true = true ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ ! -n '' ]]
+ [[ ! -n '' ]]
+ unset noaurpkgs
+ pkgs=(${noaurpkgs[@]})
+ norepopkgs=($($pacmanbin -Sp "${noaurpkgs[@]}" 2>&1 >/dev/null | awk -F " " '{print $NF}'))
++ pacman -Sp
++ awk -F ' ' '{print $NF}'
+ for i in '"${norepopkgs[@]}"'
+ [[ !    =~ [a-zA-Z0-9.+-]+/help)[^a-zA-Z0-9.+-] ]]
+ aurpkgs+=("$i")
+ repopkgs=($(LC_COLLATE=C comm -23 <(tr ' ' '\n' <<< ${pkgs[@]} | LC_COLLATE=C sort -u) <(tr ' ' '\n' <<< ${aurpkgs[@]} | LC_COLLATE=C sort -u)))
++ LC_COLLATE=C
++ comm -23 /dev/fd/63 /dev/fd/62
+++ tr ' ' '\n'
+++ tr ' ' '\n'
+++ LC_COLLATE=C
+++ sort -u
+++ LC_COLLATE=C
+++ sort -u
+ unset pkgs
+ [[ ! -n '' ]]
+ sudo pacman -S -y -u
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 multilib is up to date
:: Starting full system upgrade...
 there is nothing to do
+ [[ ! -n '' ]]
+ [[ -n '' ]]
+ [[ -n help) ]]
+ [[ true = true ]]
+ [[ ! -n '' ]]
+ note w 'Package(s) \e[1;39mhelp)\e[0m not found in repositories, trying \e[1;35mAUR\e[0m...'
+ case "$1" in
+ echo -e '\e[1;33m::\e[0m Package(s) \e[1;39mhelp)\e[0m not found in repositories, trying \e[1;35mAUR\e[0m...'
[1;33m::[0m Package(s) [1;39mhelp)[0m not found in repositories, trying [1;35mAUR[0m...
+ [[ ! -n '' ]]
+ [[ true = true ]]
+ Core 'help)'
+ getignoredpkgs
+ ignoredpkgs+=($(grep '^IgnorePkg' '/etc/pacman.conf' | cut -d '=' -f 2- | tr -d "'\""))
++ grep '^IgnorePkg' /etc/pacman.conf
++ cut -d = -f 2-
++ tr -d ''\''"'
+ [[ -e /home/jake/.config/cower/config ]]
+ ignoredpkgs=(${ignoredpkgs[@]//,/ })
+ [[ -n true ]]
+ UpgradeAur
+ note i '\e[1;39mStarting AUR upgrade...\e[0m'
+ case "$1" in
+ echo -e '\e[1;34m::\e[0m \e[1;39mStarting AUR upgrade...\e[0m'
[1;34m::[0m [1;39mStarting AUR upgrade...[0m
+ [[ -e /tmp/pacaurtmp-jake/ignoretmp ]]
+ rm /tmp/pacaurtmp-jake/ignoretmp
+ aurpkgs+=($(cower -u ${coweropts[@]} ${pkgs[@]} 2>"$tmpdir/ignoretmp" | sed -r "s:\x1B\[[0-9;]*[mK]::g" | awk -F " " '{print $2}'))
++ cower -u -f -f --color=auto
++ sed -r 's:\x1B\[[0-9;]*[mK]::g'
++ awk -F ' ' '{print $2}'
+ [[ -e /tmp/pacaurtmp-jake/ignoretmp ]]
+ aurpkgs+=($(awk -F " " '{print $2}' "$tmpdir/ignoretmp" | sed -r "s:\x1B\[[0-9;]*[mK]::g"))
++ awk -F ' ' '{print $2}' /tmp/pacaurtmp-jake/ignoretmp
++ sed -r 's:\x1B\[[0-9;]*[mK]::g'
+ foreignpkgs=($($pacmanbin -Qmq))
++ pacman -Qmq
+ downloadjson android-sdk android-sdk-build-tools android-sdk-platform-tools android-studio arduino armrr aura-bin binreader clamtk cower cross-mipsel-linux-gnu-binutils cross-mipsel-linux-gnu-gcc dropbox epson-alcx11-filter filebot freetype2-ubuntu gitflow google-chrome google-cloud-sdk hidapi-git hjsplit-bin ibus-mozc intellij-idea-ultimate-edition jdk lib32-at-spi2-atk lib32-at-spi2-core lib32-colord lib32-cracklib lib32-dbus-glib lib32-dconf lib32-flex lib32-gconf lib32-gtk3 lib32-js17 lib32-json-glib lib32-libappindicator lib32-libdbusmenu-glib lib32-libdbusmenu-gtk2 lib32-libgcrypt15 lib32-libgusb lib32-libindicator-gtk2 lib32-libpng12 lib32-libsoup lib32-libtirpc lib32-libxkbcommon lib32-pam lib32-polkit lib32-rest libdbusmenu-gtk2 libgcrypt15 libindicator-gtk2 libindicator-gtk3 libpng12 libudev.so.0 maven-3.0.5 megasync menda-themes-dark minecraft minted mozc nikto pacaur-git perl-file-homedir perl-text-csv powerline-fonts-git preload prezto-git pycharm-professional python-autopep8 python-pymysql python-pyusb python2-hidapi python2-yenc robomongo rstudio-desktop-bin rubymine sabnzbd shflags spotify steam-native sublime-text-dev teamviewer texworks ttf-google-fonts-git ttf-vlgothic ttf-win7-fonts-autodownload uncrustify unified-remote-server universalindentgui vertex-themes virtualbox-ext-oracle wimlib xboxdrv xz-git
+ aururl='https://aur.archlinux.org/rpc.php?type=multiinfo&v=3'
+ urlencodedpkgs=($(sed 's/+/%2b/g;s/@/%40/g' <<< $@))
++ sed 's/+/%2b/g;s/@/%40/g'
++ printf '&arg[]=%s' android-sdk android-sdk-build-tools android-sdk-platform-tools android-studio arduino armrr aura-bin binreader clamtk cower cross-mipsel-linux-gnu-binutils cross-mipsel-linux-gnu-gcc dropbox epson-alcx11-filter filebot freetype2-ubuntu gitflow google-chrome google-cloud-sdk hidapi-git hjsplit-bin ibus-mozc intellij-idea-ultimate-edition jdk lib32-at-spi2-atk lib32-at-spi2-core lib32-colord lib32-cracklib lib32-dbus-glib lib32-dconf lib32-flex lib32-gconf lib32-gtk3 lib32-js17 lib32-json-glib lib32-libappindicator lib32-libdbusmenu-glib lib32-libdbusmenu-gtk2 lib32-libgcrypt15 lib32-libgusb lib32-libindicator-gtk2 lib32-libpng12 lib32-libsoup lib32-libtirpc lib32-libxkbcommon lib32-pam lib32-polkit lib32-rest libdbusmenu-gtk2 libgcrypt15 libindicator-gtk2 libindicator-gtk3 libpng12 libudev.so.0 maven-3.0.5 megasync menda-themes-dark minecraft minted mozc nikto pacaur-git perl-file-homedir perl-text-csv powerline-fonts-git preload prezto-git pycharm-professional python-autopep8 python-pymysql python-pyusb python2-hidapi python2-yenc robomongo rstudio-desktop-bin rubymine sabnzbd shflags spotify steam-native sublime-text-dev teamviewer texworks ttf-google-fonts-git ttf-vlgothic ttf-win7-fonts-autodownload uncrustify unified-remote-server universalindentgui vertex-themes virtualbox-ext-oracle wimlib xboxdrv xz-git
+ urlargs='&arg[]=android-sdk&arg[]=android-sdk-build-tools&arg[]=android-sdk-platform-tools&arg[]=android-studio&arg[]=arduino&arg[]=armrr&arg[]=aura-bin&arg[]=binreader&arg[]=clamtk&arg[]=cower&arg[]=cross-mipsel-linux-gnu-binutils&arg[]=cross-mipsel-linux-gnu-gcc&arg[]=dropbox&arg[]=epson-alcx11-filter&arg[]=filebot&arg[]=freetype2-ubuntu&arg[]=gitflow&arg[]=google-chrome&arg[]=google-cloud-sdk&arg[]=hidapi-git&arg[]=hjsplit-bin&arg[]=ibus-mozc&arg[]=intellij-idea-ultimate-edition&arg[]=jdk&arg[]=lib32-at-spi2-atk&arg[]=lib32-at-spi2-core&arg[]=lib32-colord&arg[]=lib32-cracklib&arg[]=lib32-dbus-glib&arg[]=lib32-dconf&arg[]=lib32-flex&arg[]=lib32-gconf&arg[]=lib32-gtk3&arg[]=lib32-js17&arg[]=lib32-json-glib&arg[]=lib32-libappindicator&arg[]=lib32-libdbusmenu-glib&arg[]=lib32-libdbusmenu-gtk2&arg[]=lib32-libgcrypt15&arg[]=lib32-libgusb&arg[]=lib32-libindicator-gtk2&arg[]=lib32-libpng12&arg[]=lib32-libsoup&arg[]=lib32-libtirpc&arg[]=lib32-libxkbcommon&arg[]=lib32-pam&arg[]=lib32-polkit&arg[]=lib32-rest&arg[]=libdbusmenu-gtk2&arg[]=libgcrypt15&arg[]=libindicator-gtk2&arg[]=libindicator-gtk3&arg[]=libpng12&arg[]=libudev.so.0&arg[]=maven-3.0.5&arg[]=megasync&arg[]=menda-themes-dark&arg[]=minecraft&arg[]=minted&arg[]=mozc&arg[]=nikto&arg[]=pacaur-git&arg[]=perl-file-homedir&arg[]=perl-text-csv&arg[]=powerline-fonts-git&arg[]=preload&arg[]=prezto-git&arg[]=pycharm-professional&arg[]=python-autopep8&arg[]=python-pymysql&arg[]=python-pyusb&arg[]=python2-hidapi&arg[]=python2-yenc&arg[]=robomongo&arg[]=rstudio-desktop-bin&arg[]=rubymine&arg[]=sabnzbd&arg[]=shflags&arg[]=spotify&arg[]=steam-native&arg[]=sublime-text-dev&arg[]=teamviewer&arg[]=texworks&arg[]=ttf-google-fonts-git&arg[]=ttf-vlgothic&arg[]=ttf-win7-fonts-autodownload&arg[]=uncrustify&arg[]=unified-remote-server&arg[]=universalindentgui&arg[]=vertex-themes&arg[]=virtualbox-ext-oracle&arg[]=wimlib&arg[]=xboxdrv&arg[]=xz-git'
+ [[ 1896 -lt 8125 ]]
+ curl -sfg --compressed -C 0 'https://aur.archlinux.org/rpc.php?type=multiinfo&v=3&arg[]=android-sdk&arg[]=android-sdk-build-tools&arg[]=android-sdk-platform-tools&arg[]=android-studio&arg[]=arduino&arg[]=armrr&arg[]=aura-bin&arg[]=binreader&arg[]=clamtk&arg[]=cower&arg[]=cross-mipsel-linux-gnu-binutils&arg[]=cross-mipsel-linux-gnu-gcc&arg[]=dropbox&arg[]=epson-alcx11-filter&arg[]=filebot&arg[]=freetype2-ubuntu&arg[]=gitflow&arg[]=google-chrome&arg[]=google-cloud-sdk&arg[]=hidapi-git&arg[]=hjsplit-bin&arg[]=ibus-mozc&arg[]=intellij-idea-ultimate-edition&arg[]=jdk&arg[]=lib32-at-spi2-atk&arg[]=lib32-at-spi2-core&arg[]=lib32-colord&arg[]=lib32-cracklib&arg[]=lib32-dbus-glib&arg[]=lib32-dconf&arg[]=lib32-flex&arg[]=lib32-gconf&arg[]=lib32-gtk3&arg[]=lib32-js17&arg[]=lib32-json-glib&arg[]=lib32-libappindicator&arg[]=lib32-libdbusmenu-glib&arg[]=lib32-libdbusmenu-gtk2&arg[]=lib32-libgcrypt15&arg[]=lib32-libgusb&arg[]=lib32-libindicator-gtk2&arg[]=lib32-libpng12&arg[]=lib32-libsoup&arg[]=lib32-libtirpc&arg[]=lib32-libxkbcommon&arg[]=lib32-pam&arg[]=lib32-polkit&arg[]=lib32-rest&arg[]=libdbusmenu-gtk2&arg[]=libgcrypt15&arg[]=libindicator-gtk2&arg[]=libindicator-gtk3&arg[]=libpng12&arg[]=libudev.so.0&arg[]=maven-3.0.5&arg[]=megasync&arg[]=menda-themes-dark&arg[]=minecraft&arg[]=minted&arg[]=mozc&arg[]=nikto&arg[]=pacaur-git&arg[]=perl-file-homedir&arg[]=perl-text-csv&arg[]=powerline-fonts-git&arg[]=preload&arg[]=prezto-git&arg[]=pycharm-professional&arg[]=python-autopep8&arg[]=python-pymysql&arg[]=python-pyusb&arg[]=python2-hidapi&arg[]=python2-yenc&arg[]=robomongo&arg[]=rstudio-desktop-bin&arg[]=rubymine&arg[]=sabnzbd&arg[]=shflags&arg[]=spotify&arg[]=steam-native&arg[]=sublime-text-dev&arg[]=teamviewer&arg[]=texworks&arg[]=ttf-google-fonts-git&arg[]=ttf-vlgothic&arg[]=ttf-win7-fonts-autodownload&arg[]=uncrustify&arg[]=unified-remote-server&arg[]=universalindentgui&arg[]=vertex-themes&arg[]=virtualbox-ext-oracle&arg[]=wimlib&arg[]=xboxdrv&arg[]=xz-git' -o /tmp/pacaurtmp-jake/rpc.json
+ allaurpkgs=($(getjsonvar "Name"))
++ getjsonvar Name
++ json_reformat
++ tr -d '", '
++ grep -Po 'Name:.*'
++ awk -F '#' '{print $2}'
++ sed -r s/Name:/Name#/g
+ aurforeignpkgs=($(LC_COLLATE=C comm -23 <(tr ' ' '\n' <<< ${foreignpkgs[@]} | LC_COLLATE=C sort -u) <(tr ' ' '\n' <<< ${allaurpkgs[@]} | LC_COLLATE=C sort -u)))
++ LC_COLLATE=C
++ comm -23 /dev/fd/63 /dev/fd/62
+++ tr ' ' '\n'
+++ LC_COLLATE=C
+++ sort -u
+++ tr ' ' '\n'
+++ LC_COLLATE=C
+++ sort -u
+ [[ -n '' ]]
+ aurpkgs=($(tr ' ' '\n' <<< ${aurpkgs[@]} | sort -u))
++ tr ' ' '\n'
++ sort -u
+ nothingtodo 'help)'
+ [[ -z help) ]]
+ return 0
+ IgnoreChecks
+ [[ -z '' ]]
+ return
+ DepsSolver
+ note i 'resolving dependencies...'
+ case "$1" in
+ echo -e '\e[1;34m::\e[0m resolving dependencies...'
[1;34m::[0m resolving dependencies...
+ (( i=0 ))
+ (( i<1 ))
++ awk -F '>|<|=' '{print $1}'
+ aurpkgs[$i]='help)'
+ (( i++ ))
+ (( i<1 ))
+ deps+=("${aurpkgs[@]}")
+ FindDepsAur 'help)'
+ [[ -n '' ]]
+ [[ -z '' ]]
+ depspkgsaur=("${aurpkgs[@]}")
+ downloadjson 'help)'
+ aururl='https://aur.archlinux.org/rpc.php?type=multiinfo&v=3'
+ urlencodedpkgs=($(sed 's/+/%2b/g;s/@/%40/g' <<< $@))
++ sed 's/+/%2b/g;s/@/%40/g'
++ printf '&arg[]=%s' 'help)'
+ urlargs='&arg[]=help)'
+ [[ 12 -lt 8125 ]]
+ curl -sfg --compressed -C 0 'https://aur.archlinux.org/rpc.php?type=multiinfo&v=3&arg[]=help)' -o /tmp/pacaurtmp-jake/rpc.json
+ [[ -z '' ]]
+ errdeps+=($(LC_COLLATE=C comm -3 <(tr ' ' '\n' <<< $(getjsonvar "Name") | sort -u) <(tr ' ' '\n' <<< ${aurpkgs[@]} | sort -u)))
++ LC_COLLATE=C
++ comm -3 /dev/fd/63 /dev/fd/62
+++ tr ' ' '\n'
+++ sort -u
+++ tr ' ' '\n'
+++ sort -u
++++ getjsonvar Name
++++ json_reformat
++++ tr -d '", '
++++ grep -Po 'Name:.*'
++++ sed -r s/Name:/Name#/g
++++ awk -F '#' '{print $2}'
+ [[ -n '' ]]
+ depspkgs=($(getjsonarray "Depends"))
++ getjsonarray Depends
++ json_reformat
++ tr -d ', '
++ sed -e '/^"Depends"/,/]/!d'
++ tr -d '\"'
++ tr '\n' ' '
++ sed 's/] /]\n/g'
++ tr -d '\n'
++ awk -F : '{print $2}'
++ tr -d '[]"'
+ depspkgs+=($(getjsonarray "MakeDepends"))
++ getjsonarray MakeDepends
++ json_reformat
++ tr -d ', '
++ sed -e '/^"MakeDepends"/,/]/!d'
++ tr -d '\"'
++ tr '\n' ' '
++ sed 's/] /]\n/g'
++ awk -F : '{print $2}'
++ tr -d '[]"'
++ tr -d '\n'
+ depspkgs=($($pacmanbin -T "${depspkgs[@]}" | sort -u))
++ pacman -T
++ sort -u
+ unset depspkgsaur
+ [[ -n '' ]]
+ unset depspkgs
+ [[ -n '' ]]
+ [[ -n '' ]]
+ downloadjson 'help)'
+ aururl='https://aur.archlinux.org/rpc.php?type=multiinfo&v=3'
+ urlencodedpkgs=($(sed 's/+/%2b/g;s/@/%40/g' <<< $@))
++ sed 's/+/%2b/g;s/@/%40/g'
++ printf '&arg[]=%s' 'help)'
+ urlargs='&arg[]=help)'
+ [[ 12 -lt 8125 ]]
+ curl -sfg --compressed -C 0 'https://aur.archlinux.org/rpc.php?type=multiinfo&v=3&arg[]=help)' -o /tmp/pacaurtmp-jake/rpc.json
+ depsAname=($(getjsonvar "Name"))
++ getjsonvar Name
++ json_reformat
++ tr -d '", '
++ grep -Po 'Name:.*'
++ awk -F '#' '{print $2}'
++ sed -r s/Name:/Name#/g
+ depsAver=($(getjsonvar "Version"))
++ getjsonvar Version
++ json_reformat
++ tr -d '", '
++ grep -Po 'Version:.*'
++ awk -F '#' '{print $2}'
++ sed -r s/Version:/Version#/g
+ depsAood=($(getjsonvar "OutOfDate"))
++ getjsonvar OutOfDate
++ json_reformat
++ tr -d '", '
++ grep -Po 'OutOfDate:.*'
++ awk -F '#' '{print $2}'
++ sed -r s/OutOfDate:/OutOfDate#/g
+ (( i=0 ))
+ (( i<0 ))
+ depsAlastmodified=($(getjsonvar "LastModified"))
++ getjsonvar LastModified
++ json_reformat
++ tr -d '", '
++ grep -Po 'LastModified:.*'
++ awk -F '#' '{print $2}'
++ sed -r s/LastModified:/LastModified#/g
+ (( i=0 ))
+ (( i<0 ))
+ for i in '"${errdeps[@]}"'
+ note f 'no results found for help)'
+ case "$1" in
+ echo -e '\e[1;31m::\e[0m no results found for help)'
[1;31m::[0m no results found for help)
+ [[ -n help) ]]
+ exit 1
```

**Comment by rmarquis** [comment]
Thanks for the report. This is likely a regression introduced in the very last commit. I'll have a look today.

**Comment by rmarquis** [comment]
Quick fix pushed in df6ebf35965fb4edcdf48b9bf7d1f2198d2b151a. I'll take time today to do more test to ensure all possible user cases are still handled correctly.
